bcda/client/bluebutton.go

```diff
@@ -182,14 +182,16 @@ func addRequestHeaders(req *http.Request, reqID uuid.UUID) {
 	req.Header.Add("BlueButton-OriginalUrl", req.URL.String())
 	req.Header.Add("BlueButton-OriginalQuery", req.URL.RawQuery)
 	req.Header.Add("BlueButton-BackendCall", "")
+	req.Header.Add("BCDA-JOBID", jobID)
+	req.Header.Add("BCDA-CMSID", cmsID)
 }
 
 func logRequest(req *http.Request, jobID string) {
```

Comment:
Can we drop the `jobID` parameter from `logRequest()` and `logResponse()` now that we're getting it from the header?

Reply:
yessss. Again you got the eyes